Core/clim-basic/drawing/design.lisp

```diff
@@ -360,6 +364,12 @@
     (color-rgb (indirect-ink-ink +foreground-ink+))
     (opacity-value color)))
 
+;;; Regions
+(defmethod design-ink ((design region) x y)
+  (if (region-contains-position-p design x y)
+      (design-ink +foreground-ink+ x y)
+      +transparent-ink+))
```

Comment:
Is this mandated by the spec? If so, maybe cite the relevant section. I was only aware of `draw-design` for this kind of thing.

Reply:
it is, but it is quite obvious, Designs chapter has a whole section "http://bauhh.dyndns.org:8000/clim-spec/14-4.html#_745", I don't think that adding a comment is necessary.

http://bauhh.dyndns.org:8000/clim-spec/14-4.html#_745

Reply:
You are right. It is clear in the specification.